source/guides/getting-started.md

```diff
@@ -159,6 +159,16 @@ Note that, although Hanami is ready for a Behavior Driven Development workflow o
 We'll go with [Minitest](https://github.com/seattlerb/minitest) here (which is the default), but we can use [RSpec](http://rspec.info) by creating the project with `--test=rspec` option.
 Hanami will then generate helpers and stub files for it.
 
+Nonetheless we'll have to setup the test database to be able to run the tests.
+First we have to define our database url in `.env.test`.
+Now our database is configured we have to create it and migrate our schema by running
+
+```shell
+% HANAMI_ENV=test bundle exec db prepare
```

Comment:
I can't find anything in the getting started guide that states we have to do `hanami create db`. In the guide it work because we make the assumption that the reader uses sqlite.

So I don't think we've already created the database. Do we have to be explicit about that and add something saying that if you're using a "real" DB, you have to create the DB?

And even if the user has created the DB, I think that only the development one has been created, he still has to create the test one or do I miss something?